### PR TITLE
path fix in GUI

### PR DESCRIPTION
**Pull request by austin-millan**
path fix: `example_config.json` is relative to RunManager's `__file__` , aka: (/ENTIRE/ABS/PATH/TO/run_manager.py)

### CODE REVIEW

File: gui/run_manager.py [hunk 1]
```diff
@@ -355,10 +355,9 @@ def compare_tags(self, a, b):
...
-
-    run_manager.update_run(tag_to_find="TAG",
-                           data=new_args)
\ No newline at end of file
+    # new_args = {"tag": "NEW TAG NAME",
+    #             "args": {
+    #                 "Kit Version": "NEW KIT VERSION"}}
+    #
+    # run_manager.update_run(tag_to_find="TAG",
+    #                        data=new_args)
```
Comment (benjspriggs) [inline gui/run_manager.py]: We should remove unused code where possible.

Comment (lenhattu) [reply gui/run_manager.py]: that should be another issue named "clean code + reformat"

Comment (benjspriggs) [reply gui/run_manager.py]: @lenhattu I disagree, but either will work since we're freezing new features.

Comment (austin-millan) [reply gui/run_manager.py]: Are we freezing all new features next Wed?

**Pull request merged**